packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkReactNativePackage.kt

```diff
@@ -17,11 +17,12 @@ import com.facebook.react.module.model.ReactModuleInfoProvider
  */
 class DdSdkReactNativePackage : TurboReactPackage() {
     override fun getModule(name: String, reactContext: ReactApplicationContext): NativeModule? {
+        val sdkWrapper = DatadogSDKWrapper()
```

Comment:
if they __can__ share the same instance (if I understand properly the intention for this change), maybe it is worth making `sdkWrapper` as a lazy property in this class?

Otherwise they will still have different `DatadogSDKWrapper` instances.